client/src/Components/ShippingInformation.jsx

```diff
@@ -0,0 +1,186 @@
+    this.setState({
+      shippingTime: manufacturingTime,
+      shippingFrom: shippingFrom,
+      shippingTo: shippingTo,
+      deliveryDate : deliveryDate
+    })
+  }
+
+  getReturnPolicy(){
+    var returnable;
+    var returnPolicy ="";
+    var exclusion;
+    var wrapping = "";
+    if (this.props.data[0]['availableToReturn']){
+      returnable = "Returns and exchanges accepted"
+      exclusion = "Exceptions may apply. "
+      returnPolicy ="See return policy"
+      wrapping = "Gift wrapping available"
+    } else{
+      returnable ="No returns or exchanges"
+      exclusion = "But please contact me if you have any problems with your order."
+    }
+    this.setState({
+      exclusions : exclusion,
+      returnPolicy : returnable,
+      returnOverview : returnPolicy,
+      giftWrapping : wrapping
+    })
+  }
+
+  getShippingInfo(){
+    event.preventDefault();
+    this.setState({
+      getShippingCost: !this.state.getShippingCost
+    })
+  }
+
+  showShipping(){
+    event.preventDefault();
+  }
+  
+  shippingCost(){
+    let shippingCost = this.props.data[0]['shippingCosts'];
+    if (typeof parseInt(event.target.value) === 'number' && event.target.value.length === 5){
+      this.setState({
+        shippingCost: shippingCost,
+        enteredShippingInfo: true,
+        getShippingCost: !this.state.getShippingCost,
+        zipCode: event.target.value
+      })
+    }
+  }
+
+  shippingToCountry (){
+    this.setState({
+      shippingToCountry: event.target.value
+    })
+  }
+
+  render(){
+    return(
+    <div>
+    <div className="shippingInformation">
+      <div style={{fontWeight: 'bold'}}>
+        Shipping & Returns
+      </div>
+      {!this.state.enteredShippingInfo ?
+        <div>
+          <div className="manufacturingTime" style={{fontWeight: 'bold'}}>
+            Ready to ship in {this.state.shippingTime}-{this.state.shippingTime + 2} days
+          </div>
+          <div className="shippedFrom">
+            From {this.state.shippingFrom}
+          </div>
+          <div className="getShippingCost" style={{textDecorationLine: 'underline', cursor: "pointer"}} onClick={this.getShippingInfo}>
+             Get Shipping Cost 
+          </div> 
+        </div> :
+
+        <div> 
+          <div className='deliver' style={{fontWeight: 'bold'}}>
+            Estimated delivery: {this.state.deliveryDate[0]} 4 - {this.state.deliveryDate[1]} 7
+          </div>
+          <div className="shippedFrom">
+            From {this.state.shippingFrom}
+          </div>
+          <span className="shippingCost">
+            ${this.state.shippingCost} shipping to 
+          </span>      
+          <span className="shippingAddress" 
+            style={{textDecorationLine: 'underline'}}
+            onClick={this.getShippingInfo}
+          >
+          {this.state.shippingToCountry}, {this.state.zipCode}
+          </span>      
+        </div>
+      }
+
+          {this.state.getShippingCost ?
```

Comment:
same comment; only use ternary where it is easy to read, generally recommended for if it can be done on a single line